packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  Fragment,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-link',
+  styleUrl: 'va-link.css',
+  shadow: true,
+})
+export class VaLink {
+  /**
+   * If `true`, the anchor text will be bolded and include a right arrow icon.
+   */
+  @Prop() active?: boolean = false;
```

Comment:
I'm a bit confused about this prop. Is it meant to be different than the notion of `:active` in CSS?

Reply:
This prop is meant to be used when the [Active link variant](https://design.va.gov/components/link/#active-link) is required.